app/templates/admin/createEvent.html

```diff
@@ -152,17 +154,58 @@ <h1 id="title">{{page_title}}</h1>
           </div>
           <div class='form-group'>
             <label class="form-label" for='inputEventFacilitators'><strong>Facilitators</strong></label>
-              <select class="form-select" multiple id="inputEventFacilitators" name="facilitators" >
-              <!-- make sure we show existing facilitators in case they aren't in the facilitators list anymore -->
-              {% for ef in eventData.facilitators %}
-                {% if ef not in allFacilitators %}
-                  <option value="{{ef.username}}" selected>{{ef.firstName}} {{ef.lastName}}</option>
-                {% endif %}
-              {% endfor %}
-              {% for f in allFacilitators %}
-                <option value="{{f.username}}" {{ "selected" if f in eventData.facilitators }}> {{f.firstName}} {{f.lastName}}</option>
-              {% endfor %}
-              </select>
+
+
+            {% if isEditable %}
+            <input class="form-control" type="text" id="hiddenFacilitatorArray" name="facilitators" value="" hidden>
+            <input
+              id="eventFacilitator"
+              class="form-control"
+              type="search"
+              placeholder="Search facilitator"
+              value="" />
+            <table
+              class="table mb-3"
+              id="facilitatorTable"
+              name="facilitatorTable">
+              <thead>
+                <tr>
+                  <th scope="col">Facilitator</th>
+                  <th scope="col">Remove</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr hidden>
+                  <td>
+                    <p class="facilitatorName" required></p>
+                  </td>
+                  <td><button id="remove" type="button" class="btn btn-danger">Remove</button></td>
+
+                </tr>
+                {% for instructor in courseInstructor %}
+                <tr>
+                  <td>
+                    <p class="facilitatorName">{{instructor.user.firstName}} {{instructor.user.lastName}} ({{instructor.user}})</p>
+                  </td>
+                  <td>
+                    <button id="remove" type="button" class="btn btn-danger">Remove</button>
+                  </td>
+                </tr>
+                {% endfor %}
+              </tbody>
+            </table>
+            {% else %}
+              {% if not eventData.facilitators|count %}
+                <p>None</p>
+                {%endif%}
+                {% for f in eventData.facilitators %}
+                <p>{{f.firstName}} {{f.lastName}}</p>
+                {% endfor %}
```

Comment:
This bit should go in the view template

Reply:
This part should be kept here for when the event edited. If there are facilitators, the person editing the event should be able to see them. However, this currently does not work with the new search bar, but I think Mostaphe created an issue for it to keep this PR from getting overly large.

Reply:
This page is always editable. That's why there aren't any more isEditable references